messaging/messaging/pom.xml

```diff
@@ -60,6 +60,14 @@
       <groupId>org.wildfly.swarm</groupId>
       <artifactId>security</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.wildfly.swarm</groupId>
+      <artifactId>jgroups</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.wildfly.swarm</groupId>
```

Comment:
Same remark than above (the Messaging subsystem depends on Undertow to multiplex its messaging protocol on top of HTTP)